mdagent/subagents/subagent_setup.py

```diff
@@ -12,17 +12,18 @@ def __init__(
         temp=0.1,
         max_iterations=40,
         verbose=True,
-        ckpt_dir="ckpt",
         resume=False,
         retrieval_top_k=5,
         curriculum=True,
     ):
         self.path_registry = path_registry
+        if self.path_registry is None:
```

Comment:
how about raising an error instead? it would be problematic if the sub agents have a different checkpoint, than the other tools right?

Reply:
AH, get instance gets the current ckpt, it doesn't make a new one :)